project/src/main/java/com/google/sps/data/Place.java

```diff
@@ -52,44 +52,95 @@
   public abstract String phone();
 
   /**
-   * @return The rating of the place, represented by a number between 1-5.
+   * @return the rating of the place, represented by a number between 1-5.
    */
-  public abstract int rating();
+  public abstract double rating();
```

Comment:
The Google Place API returns the rating for each place as float

Reply:
changed it, thanks!